### ISSUE TITLE
[Feature request] whitelist

### ISSUE THREAD

**Issue by CSchulz** [christopherthielen/check-peer-dependencies#12]
Sometimes you have to keep some dependencies unresolved because your project setup does not allow another way f.e.:

```
  ✅  karma * is required by karma-jasmine@4.0.1 (5.2.3 is installed)
  ✅  karma >=0.9 is required by karma-jasmine-html-reporter@1.5.4 (5.2.3 is installed)
  ✅  karma >=0.9 is required by karma-junit-reporter@1.2.0 (5.2.3 is installed)
  ✅  karma >=0.9 is required by karma-sabarivka-reporter@3.2.5 (5.2.3 is installed)
  ❌  tslib ^1.10.0 is required by @angular/core@9.0.0) (2.0.3 is installed)
  ❌  tslib ^1.10.0 is required by @angular/compiler@9.0.0) (2.0.3 is installed)
  ✅  tslib ^2.0.0 is required by ng-packagr@11.0.3 (2.0.3 is installed)
  ✅  tslint ^6.1.0 is required by @angular-devkit/build-angular@0.1100.7 (6.1.3 is installed)
  ✅  tslint ^5.0.0 || ^6.0.0 is required by codelyzer@6.0.1 (6.1.3 is installed)
  ❌  tslint ^5.0.0 is required by tslint-consistent-codestyle@1.16.0) (6.1.3 is installed)
  ❌  tslint ^5.0.0 is required by @fimbul/bifrost@0.21.0) (6.1.3 is installed)

Searching for solutions for 4 missing dependencies...

  ❌  Unable to find a version of tslib that satisfies the following peerDependencies: ^1.10.0 and ^2.0.0
  ❌  Unable to find a version of tslint that satisfies the following peerDependencies: ^6.1.0 and ^5.0.0 || ^6.0.0 and ^5.0.0
```

I would like to whitelist some packages to use this tool to check the rest of our project.

**Comment by Josmithr** [comment]
I would love to see this as well. I have a couple of repos I would love to integrate this tool into, but which is currently blocked by known peer dep violations we are stuck with (at least for the time being)

**Comment by christopherthielen** [comment]
Fixed in https://github.com/christopherthielen/check-peer-dependencies/commit/5994c9c9e41388a7d66b1615a50c098f7e8edf0f and released as 4.1.0

Added as `--ignore foo --ignore bar` cli option.  Also added the ability to store these (and any other config) in package.json:

```json
...
  "checkPeerDependencies": ["ignorepkg1", "ignorepkg2"]
...
```

**Comment by christopherthielen** [comment]
@CSchulz @Josmithr FYI
